src/pages/Trainee/Trainee.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { AddDialog } from './component';
+
+const Trainee = () => (
+  <AddDialog />
```

Comment:
AddDialog component should receive props.
https://redmine.successive.tech/issues/40835
Please check point: 3